assets/lokomotive-kubernetes/aws/flatcar-linux/kubernetes/aws-ebs-csi-driver/Chart.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
```

Comment:
So from what I read in the commit message you didn't take the upstream Helm chart, you took its metadata and then used some plain YAML files. Where did you take those from?

I think it might be worth finding out why the upstream Helm chart didn't work for you and fixing that so we can reuse it and potentially expose some of its knobs to the users in the future if needed.

Reply:
Regarding where the plain YAML files come from, they're from the version of the files around when you went through the example as mentioned in #379 : https://github.com/kubernetes-sigs/aws-ebs-csi-driver/tree/72f9a0940eb46e06704ccd81a6070c6f317e79d5/deploy/kubernetes/base